app/scripts/views/record/add-edit-controller.js

```diff
@@ -242,8 +242,6 @@
                     collapsed: true,
                     theme: 'bootstrap3',
                     iconlib: 'bootstrap3',
-                    show_errors: 'change',
-                    no_additional_properties: true,
```

Comment:
This change helps fix two new errors that appeared when upgrading JSONEditor:

- In the new version of JSONEditor, setting `show_errors: change` makes the form validate immediately on page load, which has the undesirable feature of showing errors before the user has entered any data. The default setting seems to work fine.
- In the new version of JSONEditor, the form shows `additionalProperties` errors on load. I couldn't see a reason why we would disallow `additionalProperties`, so I removed it from the forms.